utils/ExampleRunner/example-info.json

```diff
@@ -630,6 +630,10 @@
       "volumeProgressive": {
         "name": "Volume Progressive Loading",
         "description": "Volume progressive loading inter and intra image"
+      },
+      "VolumeEnhancedLoading": {
```

Comment:
DecimatedVolumeLoading or something

Reply:
hmmm did i say that? feel wrong, it is doing decimation so it should be decimatedVolumeLoading or subResolutionVolumeLoading or similiar